tests/models/t5gemma/test_modeling_t5gemma.py

```diff
@@ -53,9 +53,9 @@ class T5GemmaModelTester:
 
     if is_torch_available():
         model_class = T5GemmaModel
-        for_causal_lm_class = T5GemmaForConditionalGeneration
-        for_sequence_class = T5GemmaForSequenceClassification
-        for_token_class = T5GemmaForTokenClassification
+        causal_lm_class = T5GemmaForConditionalGeneration
+        sequence_classification_class = T5GemmaForSequenceClassification
+        token_classification_class = T5GemmaForTokenClassification
```

Comment:
this class doesn't inherit from `CausalLMTester`... but let's use the same naming conventions